tests/functional/base_test.go

```diff
@@ -92,6 +104,25 @@ func GetDefaultPlacementAPISpec() map[string]interface{} {
 	}
 }
 
+func GetTLSPlacementAPISpec() map[string]interface{} {
+	return map[string]interface{}{
+		"databaseInstance": "openstack",
+		"replicas":         1,
+		"secret":           SecretName,
+		"tls": map[string]interface{}{
+			"api": map[string]interface{}{
+				"internal": map[string]interface{}{
+					"secretName": InternalCertSecretName,
```

Comment:
you can use `names.InternalCertSecretName` and the other refs from the Names struct here. There is always a global names variable with the naming constants grouped together (it is created in suite_test). If you use `names` here then you can inline the test global `InternalCertSecretName` into CreateNames(). I know that this sounds like meaningless busy work and for a true constant like how InternalCertSecretName is defined now it probably is. The reason we have the Names struct is to encapsulate generated names like Role and RoleBinding. (nova-operator has a bunch more examples) to avoid replicating the generation logic across multiple test case.

Anyhow I let you decide which way to go as it is not a big deal.

Reply:
I see, thanks for the explanation! Will edit it, to preserve same logic within tests.